Tasks/ANTPreview/task.json

```diff
@@ -0,0 +1,168 @@
+{
+    "id": "18033022-9747-4b7b-8395-4fd7eb2b565a",
+    "name": "AntCC",
+    "friendlyName": "Ant With CodeCoverage",
```

Comment:
dont you want to mark the name with "preview" tag too? will get confusing for a user who enables preview tasks to see two tasks with same name?